build.sbt

```diff
@@ -57,9 +57,9 @@ val AkkaPersistenceCassandraVersion = "0.102"
 val PrometheusClientVersion = "0.6.0"
 val ScalaTestVersion = "3.0.5"
 val ProtobufVersion = "3.9.0"
-val GraalVersion = "19.3.0"
-val DockerBaseImageVersion = "openjdk:8-jre-slim-buster"
-val svmGroupId = if (GraalVersion startsWith "19.2") "com.oracle.substratevm" else "org.graalvm.nativeimage"
+val GraalVersion = "20.0.0"
+
+val DockerBaseImageVersion = "openjdk:8-jdk-slim-buster"
```

Comment:
@srikanthops At least I got rid of the nullSession NoSuchFieldError, but I'm not sure this is the image we should be using. What do you advise?

Reply:
[Travis CI Tests](https://travis-ci.com/github/cloudstateio/cloudstate/builds/152907832) passed with my [original pull request](https://github.com/cloudstateio/cloudstate/pull/207)

this image addresses/fixes a few vulnerabilities